src/controllers/login.js

```diff
@@ -1,27 +1,31 @@
 import {isAValidPhoneNumber, isAValidUserType} from '../util/validator';
 import {errorMessages} from '../util/constants';
-import {getUsersByPhoneNumberAndUserType, registerNewUser} from '../services/users';
+import {createUser, getUserById, getUsersByPhoneNumberAndUserType} from '../services/users';
 
 export const loginRoute = {
     path: '/login',
     method: 'GET',
     options: {
-        handler: async (request, h) => {
+        handler: async (request) => {
             const phoneNumber = request.query.phoneNumber;
             const userType = request.query.userType;
+            userType.toLowerCase();
```

Comment:
💡 🤔   it wont change `userType` to `lowercase`, it returns another string in `lowercase` of `userType` .